.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - 4
```

Comment:
um, why is node 4 dropped? eslint still supports node 4; this makes it a breaking change.